Input documentation/Test Model/ModelInfo.json

```diff
@@ -3,17 +3,17 @@
     Layers: [
         {
             Caption: "Body",
-            Simulation: false,
+            DataType: "anatomy",
             Directory: "test_body"

```

Comment:
note that this is not a valid json file as it has no quotes on keys

Reply:
Good catch. That's my fault originally :), as I introduced these JSON files and in all my samples I didn't use quotes. Reading https://stackoverflow.com/questions/949449/do-the-json-keys-have-to-be-surrounded-by-quotes , quotes are indeed required by JSON on all keys (to avoid possibility of confusion with JS keywords, it seems).

Reply:
I'll fix this in HOL-44.